docs/reuse/common-blocks.md

```diff
@@ -0,0 +1,44 @@
+Start: Install Pebble binary
+
+1. Visit the [latest release page](https://github.com/canonical/pebble/releases/latest) to get the latest tag.
```

Comment:
Because just visiting the page won't "get" the tag. You have to know what you're looking for and find it, so slight wording tweak:

```suggestion
1. Visit the [latest release page](https://github.com/canonical/pebble/releases/latest) to determine the latest tag, for example, `v1.12.0`.
```

Reply:
In the previous discussion with Anne, we changed "find" to "get", but I do agree that although it's the latest release page instead of the page containing multiple releases, you'd still need to kind of "find" it, it's not like curling the URL then get the tag alone as a response.

I think the suggested modification is indeed better, I will rephrase it.